README.md

```diff
@@ -76,6 +76,10 @@ class HomepageTest < Test::Unit::TestCase
     # parameters, so make sure that `json` below is already a JSON-serialized string.
     post(uri, json, { 'CONTENT_TYPE' => 'application/json' })
   end
+  
+  def delete_with_url_params_and_body
+    delete '/?foo=bar', JSON.generate('foo' => 'baz')
+  end
```

Comment:
This example is a bit messy. Using `foo` both for the query parameter _and_ the body value, that would be a bit messy on the receiving end.

Could you change it to be `'baz' => 'zot'` instead? That would make the example more clear.